src/main/java/com/keybox/manage/action/UsersAction.java

```diff
@@ -121,17 +121,21 @@ public void validateSaveUser() {
                 || user.getFirstNm().trim().equals("")) {
             addFieldError("user.firstNm", REQUIRED);
         }
-        
-        if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
-            
-            if(!user.getPassword().equals(user.getPasswordConfirm())) {
+
+        // Password validation is only for users who are permitted to log in (ie non-passive ones)
+        if (user != null && !User.PASSIVE.equals(user.getUserType())) {
+            if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
```

Comment:
Hmm, think I'm confused. Saving a passive user's data will always have a user entity with an empty password - so [this line](https://github.com/richsage/KeyBox/blob/f5d0c6a7af6f0031eeec18537e296062c4d74cf3/src/main/java/com/keybox/manage/action/UsersAction.java#L71) would be effected?